tests/core/core.js

```diff
@@ -3,6 +3,11 @@ import SearchConfig from '../../src/core/models/searchconfig';
 import Storage from '../../src/core/storage/storage';
 import StorageKeys from '../../src/core/storage/storagekeys';
 
+jest.mock('../../src/core/utils/uuid');
+const foo = require('../../src/core/utils/uuid');
```

Comment:
does importing generateUUID like 
```js
const { generateUUID } = require('../../src/core/utils/uuid')
```
work? If that doesn't work we should rename `foo` to `uuid`